blocks/feed/feed.js

```diff
@@ -141,6 +143,11 @@ export default async function decorate(block) {
   block.innerHTML = '';
   const blockContents = resultParsers[blockType](results, blockCfg);
   const builtBlock = buildBlock(blockType, blockContents);
+  if (feedTags && blockContents.length && blockContents[0][0]) {
```

Comment:
Nit: could `blockContents[0][0]` be added to a variable as well? I see it's the images from the next line but just to have more clarity

Reply:
@dnbute Addressed it.